.github/workflows/pipeline.yml

```
@@ -0,0 +1,51 @@
+name: CarBuddy CI/CD Pipeline
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)